submissions/best-of-its-genre-nanogamer7/origins.md

```diff
@@ -0,0 +1,46 @@
+
+Wube's online presents, while there, is spread out across the forums, reddit, discord, and a few more, which makes it difficult to search for anything, really. But after some clever searching and sifting through a few interviews, I finally found  something: An [old forum thread](https://forums.factorio.com/viewtopic.php?f=5&t=3026) from March 2014, confirming Buildcraft (or IndustrialCraft, according to a different interview) as the main inspiration to Factorio.[^5][^6] While I had hoped to find an answer, I for sure wasn't expecting to find such a direct one. Having that out of the way, though, we can definitely speculate a bit more on brought Factorio to were it's at today.
+
+### Great games design themselves
+
+There is a principle in game design you should follow, that goes along the lines "follow the fun".[^7] While examples for that statement often include dramatic changes like Ape Out went through,[^8] it still applies to games that don't shift that drastically in direction. Let's take Factorio itself as an example: <!-- The example should be commented by devs, and an actually relevant feature | Options I'm considering: Pickaxe, Character Tab --> Originally the game had multiple pickaxes - a wooden, which you could then craft into a better one out of stone, and then into the iron pickaxe we had for such a long time [^citation needed]. This was clearly inspired by Minecraft (and also confirmed by the devs[^9]), but having so many items about manual mining in a game about automation gives the idea about the game - it draws the fun away. So they simplified it, gave you just your iron and steel pickaxe, but that's still two items specifically about manual mining. Thus they removed pickaxes completely from the game in `0.17.0`[^10], and with all focus on the unfun, manual labor.
+
+So once they core gameplay concept of a game is clear - which is what Factorio started out with - it's just a manner of fitting features to it. Automation? That sounds like the industrial revolution, when they invented belts. Need something to get that thing of the belts? How about inserters, they are at least not human. You get the point.
+
+<!-- Need to develop the message I'm trying to convey here a bit more -->
+
+### Conlusio
+
+While I originally tried to find Factorios "predecessor", it stemming itself with only  the core gameplay loop as an external influence makes this conclusion even more rewarding. That being said, there have probably been more automation games in the past, and Factorio definitely had a big influence on some more recent games. Maybe we can explore this further in a future article, maybe even how Factorio inspired more than just game. Who knows.
+
+
+
+---
+
+### Sources:
```

Comment:
Personally I like simple links more in the text. I never like scrolling down to see more info, and dont like the look of a list of references in an easy-reading article. But it's totally up to you, not that big of a deal.